src/components/Vizmapper/Forms/VisualPropertyValueForm.tsx

```diff
@@ -114,6 +115,253 @@ const vpType2RenderMap: Record<
   },
 }
 
+const vpType2RenderMap2: Record<
```

Comment:
These values all seem very similar.  Can you elaborate on why we need to make multiple copies with a few differences between them?